packages/next/src/build/swc/index.ts

```diff
@@ -1130,12 +1133,61 @@ function bindingToApi(
     }
   }
 
+  /**
+   * Returns a new copy of next.js config object to avoid mutating the original.
+   *
+   * Also it does some augmentation to the configuration as well, for example set the
+   * turbopack's rules if `experimental.reactCompilerOptions` is set.
+   */
+  function augmentNextConfig(
+    nextConfig: NextConfigComplete,
+    projectPath: string
+  ): Record<string, any> {
+    let ret = { ...(nextConfig as any) }
```

Comment:
nit: would prefer something like `originalNextConfig` for the argument and `nextConfig` for this variable here

Reply:
> why any

this is same as before, there are some differences to augmented config we pass to turbopack so it is not fully compatible.

Reply:
Yeah i guess that's fair